engine/hatchery/openstack/openstack.go

```diff
@@ -157,6 +157,15 @@ func (h *HatcheryOpenstack) WorkerModelsEnabled() ([]sdk.Model, error) {
 // CanSpawn return wether or not hatchery can spawn model
 // requirements are not supported
 func (h *HatcheryOpenstack) CanSpawn(ctx context.Context, model *sdk.Model, jobID int64, requirements []sdk.Requirement) bool {
+	// if there is a model, we have to check if the flavor attached to model is knowned by this hatchery
+	if model != nil {
+		if _, err := h.flavorID(model.ModelVirtualMachine.Flavor); err != nil {
+			log.Debug("CanSpawn> flavor '%s' not found", model.ModelVirtualMachine.Flavor)
```

Comment:
it could be nice to check the error

Reply:
fixed

Reply:
sorry, i don't see how you are checking the error is "flavor not found"